File: app/src/main/java/com/example/macarus0/walkiewalkie/data/WalkWithDogs.java
package com.example.macarus0.walkiewalkie.data;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.Ignore;
import android.arch.persistence.room.PrimaryKey;

@Entity
public class WalkWithDogs {

    @PrimaryKey(autoGenerate = true)
    long walkWithDogsId;

    long walkId;
    long dogId;

    @Ignore
    public void setWalkId(long walkId) {
        this.walkId = walkId;
    }

    @Ignore
    public void setDogId(long dogId) {
        this.dogId = dogId;
    }

//    @Relation(parentColumn = "walkId",
//    entityColumn = "photoId")
//    public List<String> photos;

}

File: app/src/main/java/com/example/macarus0/walkiewalkie/data/WalkWithDogs.kt
package com.example.macarus0.walkiewalkie.data

import androidx.room.Entity
import androidx.room.Ignore
import androidx.room.PrimaryKey

@Entity
class WalkWithDogs {
    @PrimaryKey(autoGenerate = true)
    var walkWithDogsId : Long = 0

    var walkId: Long = 0
    var dogId: Long = 0
}
